File: Java/Math/PalindromeInteger.java
public class Solution {
	public boolean isPalindrome(int a) {
	    if(a < 0)return false;
	    int aa = a, reverse = 0;
	    while(aa > 0){
	        reverse = reverse * 10 + aa % 10;
	        aa /= 10;
	    }
	    return reverse == a;
	}
}

File: JS/Math/PalindromeInteger.js
module.exports = {
	//param A : integer
	//return an integer
	isPalindrome : function(A){
        var reverse = 0;
        var a = A;
        while(a > 0){
            reverse = 10 * reverse + a % 10;
            a = parseInt(a / 10);
        }
        return reverse == A ? 1 : 0;
	}
};
